### PR TITLE
improve customize value of viewportWidth in block preview

### PR DESCRIPTION
**Pull request by jeff-be**
## Description

https://github.com/WordPress/gutenberg/issues/26289

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by jeff-be** [comment]
https://github.com/WordPress/gutenberg/pull/26346

**Comment by youknowriad** [comment]
At first sight, this API looks useful:

 - Should we update the block registration `example` docs to mention this property?

File: packages/block-editor/src/components/block-preview/README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ A block instance (object) or a blocks array you would like to render a preview.
 * **Type:** `Int`
 * **Default:** `700`
 
+**** Modify here the documentation to have a customize value ****
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-preview/README.md:29]: What is this about?

Comment (jeff-be) [reply packages/block-editor/src/components/block-preview/README.md:29]: See this : #26346

Comment (youknowriad) [reply packages/block-editor/src/components/block-preview/README.md:29]: Should we close the current PR?

Comment (jeff-be) [reply packages/block-editor/src/components/block-preview/README.md:29]: yes

**Pull request closed without merging**